owncloudApp/src/main/java/com/owncloud/android/capabilities/viewmodel/OCCapabilityViewModel.kt

```diff
@@ -44,8 +44,8 @@ class OCCapabilityViewModel(
         localCapabilitiesDataSource = OCLocalCapabilitiesDataSource(),
         remoteCapabilitiesDataSource = OCRemoteCapabilitiesDataSource(
             OwnCloudClientManagerFactory.getDefaultSingleton().getClientFor(
-                OwnCloudAccount(account, MainApp.getAppContext()),
-                MainApp.getAppContext()
+                OwnCloudAccount(account, MainApp.appContext),
```

Comment:
Why don't use an AndroidViewModel instead of a normal ViewModel??

It's for removing MainApp.appContext... If you don't like AndroidViewModel, you can use the injected context ;-)

Reply:
Done, using injected context